### PR TITLE
(CONT-795) Puppet 8 support / Drop Puppet 6

### PR DESCRIPTION
**Pull request by LukasAud**

### CODE REVIEW

File: .rubocop_todo.yml [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+# Offense count: 1
+# Configuration parameters: Include, CustomTransform, IgnoreMethods, SpecSuffixOnly.
+# Include: **/*_spec*rb*, **/spec/**/*
+RSpec/FilePath:
+  Exclude:
+    - 'spec/acceptance/reboot_message.spec.rb'
+
+# Offense count: 22
+RSpec/MultipleExpectations:
+  Max: 3
```
Comment (jordanbreen28) [inline .rubocop_todo.yml:100]: we can add this one to the `.rubocop.yml`. 
It's one of the agreed cops in the doc :-)

File: .rubocop_todo.yml [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+# Offense count: 21
+RSpec/StubbedMock:
+  Exclude:
+    - 'spec/unit/task/init_spec.rb'
+    - 'spec/unit/type/reboot_spec.rb'
+
+# Offense count: 1
+Style/MixinUsage:
+  Exclude:
+    - 'spec/spec_helper.rb'
```
Comment (jordanbreen28) [inline .rubocop_todo.yml:131]: just a suggestion not a request - 
I opt to disable this one inline

File: .rubocop_todo.yml (hunk 1 above)
Comment (LukasAud) [reply .rubocop_todo.yml]: addressed

File: .rubocop_todo.yml (hunk 2 above)
Comment (LukasAud) [reply .rubocop_todo.yml]: addressed

**Review by jordanbreen28 — APPROVED** [review]

**Pull request merged**